optaplanner-distribution/src/main/assembly/assembly-optaplanner.xml

```diff
@@ -63,13 +63,26 @@
         <exclude>.git/**</exclude>
       </excludes>
     </fileSet>
+    <fileSet><!-- Note: going outside the module dir is bad, but it is not fetching generated files -->
+      <directory>../optaplanner-quickstarts/spring-boot-school-timetabling</directory>
+      <outputDirectory>quickstarts/spring-boot-school-timetabling</outputDirectory>
+      <excludes>
+        <exclude>target/**</exclude>
+        <exclude>local/**</exclude>
+        <exclude>.*/**</exclude>
+        <exclude>nbproject/**</exclude>
+        <exclude>*.ipr</exclude>
+        <exclude>*.iws</exclude>
+        <exclude>*.iml</exclude>
+        <exclude>.git/**</exclude>
+      </excludes>
+    </fileSet>
```

Comment:
Instead of adding the quickstart's jar and sources-jar, we include the "build from source" sources instead. This has 2 advantages:
- users can get started with the quickstart
- we don't need to depend on artifacts from the quickstart, which causes issues when productized and when pushed to maven central (we don't own groupId "com.examples".